adafruit_displayio_layout/layouts/grid_layout.py

```diff
@@ -0,0 +1,106 @@
+# SPDX-FileCopyrightText: 2020 Kevin Matocha, Tim Cocks
+#
+# SPDX-License-Identifier: MIT
+
+"""
+`grid_layout`
+================================================================================
+
+A layout that organizes children into a grid table structure.
+
+
+* Author(s): Kevin Matocha, Tim Cocks
+
+Implementation Notes
+--------------------
+
+**Hardware:**
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+
+"""
+
+import displayio
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_DisplayIO_Layout.git"
+
+
+class GridLayout(displayio.Group):
+
+    """
+    A layout that organizes children into a grid table structure.
+
+    :param int x: x location the layout should be placed
+    :param int y: y location the layout should be placed
+    """
+
+    # pylint: disable=too-many-arguments
+    def __init__(self, x, y, width, height, grid_size, child_padding, max_children=4):
+        super().__init__(x=x, y=y, max_size=max_children)
+        self.x = x
+        self.y = y
+        self._width = width
+        self._height = height
+        self.grid_size = grid_size
+        self.child_padding = child_padding
```

Comment:
We could consider using `cells/cell` instead of `children/child`. It would eliminate the parent/child relationship you were attempting to avoid in the first place.

Reply:
I like it, will change references to child / children to cell. Thank you!